docs/services/slam/cartographer/_index.md

```diff
@@ -34,318 +22,467 @@ The source code for this module is available on the [`viam-cartographer` GitHub
 
-1. Navigate to the **Config** tab of your robot's page in [the Viam app](https://app.viam.com).
-1. Click on the **Services** subtab and click **Create service** in the lower-left corner.
-1. Select **SLAM**, then select `cartographer`.
-   You can also search for "cartographer".
-1. Click **Add module**, give your service a name of your choice, then click **Create**.
-1. In the resulting `SLAM` service configuration pane, from the `"Mapping mode"` dropdown, choose `"Create new map"`
-1. Configure the rest of the **Attributes** as follows:
+{{% /alert %}}
 
-   - `"Camera"`: Select the `name` of the camera component that you created when you [added the `rplidar` module to your robot](/modular-resources/examples/rplidar/). Example: "my-rplidar"
-     - Then set a `"Data capture rate (Hz)"` for it. Example: "5"
-   - `"Movement Sensor (Optional)"`: Select the `name` of a movement sensor component that implements the `GetAngularVelocity` and `GetLinearAcceleration` methods of the movement sensor API. Example: "my-imu"
-     - Then set a `"Data capture rate (Hz)"` for it. Example: "20"
-   - `"Minimum range (meters)"`: Set the minimum range of your `rplidar`. See [config params](#config_params) for suggested values for RPLidar A1 and A3.
-   - `"Maximum range (meters)"`: Set the maximum range of your `rplidar`. See [config params](#config_params) for suggested values for RPLidar A1 and A3.
+### Localize only
 
-If you would like to tune additional Cartographer parameters, you can expand `"Show additional parameters"`. See the [config_params](#config_params) section for more information on the other parameters.
+In this mode, the `cartographer` module on your robot executes the Cartographer algorithm itself locally to find its position on a map.
 
-To save your changes, click **Save config** at the bottom of the page.
+1.  Configure your `cartographer` SLAM service
 
-Check the **Logs** tab of your robot in the Viam app to make sure your RPlidar has connected and no errors are being raised.
+    {{< tabs name="Localize only">}}
+    {{% tab name="Config Builder" %}}
 
-{{%expand "Click to view an example JSON configuration for creating a new map" %}}
+The configuration is similar to the configuration for [updating an existing map](#update-an-existing-map), except instead of configuring a `Data capture rate (Hz)` on the camera and movement sensor, set a `Data polling rate (Hz)` on both.
+The `cartographer` module on your robot polls the live LiDAR and IMU directly at these rates, whereas the capture rate is only used when data is being sent to the cloud.
 
-See [Attributes](#attributes) for details.
+    {{% /tab %}}
+    {{% tab name="JSON Example" %}}
 
-```json {class="line-numbers linkable-line-numbers"}
-{
-  "modules": [
-    {
-      "type": "registry",
-      "name": "viam_rplidar",
-      "module_id": "viam:rplidar",
-      "version": "0.1.14"
-    },
+    ```json {class="line-numbers linkable-line-numbers"}
```

Comment:
This formatting seems messed up when rendered:

<img width="540" alt="Screenshot 2023-10-20 at 12 09 18 PM" src="https://github.com/viamrobotics/docs/assets/1517691/e9fe060a-d9db-4bee-afe0-1a707e889447">